src/main/credentialServer/httpCredentialHelperServer.ts

```diff
@@ -121,8 +120,8 @@ export class HttpCredentialHelperServer {
       const [data, error] = await serverHelper.getRequestBody(request, MAX_REQUEST_BODY_LENGTH);
 
       if (error) {
-        console.debug(`${ path }: write back status 400, error: ${ error }`);
-        response.writeHead(400, { 'Content-Type': 'text/plain' });
+        console.debug(`${ path }: write back status 413, error: ${ error }`);
```

Comment:
This line assumes that a 413 status is the only reason `getRequestBody` would return a non-nil `error`. This may be true right now, but maybe won't be in the future?

Reply:
It is true right now, but the fix is to have `serverHelper.getRequestBody` also return the error-code.

You might want to say that if we want to use that function for a different protocol it shouldn't return an HTTP-specific error code, but the function only makes sense as an HTTP server, so it's ok.